pkgs/applications/networking/instant-messengers/discord/default.nix

```
@@ -62,10 +62,17 @@ let
   };
   package = if stdenv.isLinux then ./linux.nix else ./darwin.nix;
 
-  openasar = if withOpenASAR then callPackage ./openasar.nix { } else null;
+  # OpenAsar fails with default unzip, throwing  "lchmod (file attributes) error: Operation not supported"
+  unzipFix =
+    if stdenv.isLinux then
+      unzip.overrideAttrs (oldAttrs: {buildFlags = oldAttrs.buildFlags ++ ["LOCAL_UNZIP=-DNO_LCHMOD"];})
```

@SuperSandro2000 you seemed to participate in the thread in https://gitlab.alpinelinux.org/alpine/aports/-/issues/11221 -- do you have any better suggestions on how to handle this fix?

---

```suggestion
      unzip.overrideAttrs (oldAttrs: {
        buildFlags = oldAttrs.buildFlags ++ [ "LOCAL_UNZIP=-DNO_LCHMOD" ];
      })
```

Also we should maybe change this in unzip for all packages but that should be done in a different PR.